connector/connect/src/main/scala/org/apache/spark/sql/connect/dsl/package.scala

```diff
@@ -73,6 +73,21 @@ package object dsl {
         relation.setJoin(join).build()
       }
 
+      def sample(
```

Comment:
are all required

Reply:
are all required for the `Sample` query plan, but `df.sample` API has multiple overloads which provide default values for some parameters. Should the proto plan match query plan or DF API?

Reply:
I think we can go with the query plan for now and if we want to introduce optional behavior it's going to be easy to do because the proto supports optionality by default.